Bottom: cbebfa2b63cb7d5c56a0a3c79e43af01e9711edd
Top:    164b324260b08088b87f4df598bbecaea49579dc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-20 11:05:59 -0500

Dump all host vars at start of playbook


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 2d82aa67..769c01fe 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -1,6 +1,20 @@
 ---
 # This playbook deploys the whole application stack in this site.
 
+- name: Dump Variables
+  hosts: all
+  tags:
+    - debugging
+    - always
+  tasks:
+  - template:
+      src: templates/dump_variables
+      dest: /tmp/ansible_variables
+  - fetch:
+      src: /tmp/ansible_variables
+      dest: "{{inventory_hostname}}_ansible_variables"
+      flat: yes
+
 - name: Bacula
   hosts: bacula
   remote_user: ansibleremote
diff --git a/ansible-playbook/templates/dump_variables b/ansible-playbook/templates/dump_variables
new file mode 100644
index 00000000..97fd2e34
--- /dev/null
+++ b/ansible-playbook/templates/dump_variables
@@ -0,0 +1,3 @@
+HOSTVARS (ANSIBLE GATHERED, group_vars, host_vars) :
+
+{{ hostvars | to_yaml }}
